mbedtls/src/pkcs12/mod.rs

```diff
@@ -825,33 +991,49 @@ impl Pfx {
         Ok(decrypted)
     }
 
-    fn authsafe_decrypted_contents(&self) -> impl Iterator<Item=&SafeBag> {
-        self.authsafe.contents.iter()
-            .flat_map(|d| if let AuthenticatedSafe::Data(ref d) = d { d.0.as_slice() } else { &[] })
+    fn authsafe_decrypted_contents(&self) -> impl Iterator<Item = &SafeBag> {
```

Comment:
No changes to `fn authsafe_decrypted_contents`, `fn certificates` and `fn private_keys`. Probably due to automatic linting?

Reply:
the first commit included is a rustfmt